### DISCUSSION TITLE
iTunes support for the music tv's please?

### DISCUSSION THREAD

**Discussion by emmapodioagusti** [rocksdanister/lively#1740] [Ideas]
I feel like it'd be a really nice addition, i'm old and have my entire music library on iTunes and I'd love to see this happen

**Comment by rocksdanister** [comment]
https://github.com/ModernFlyouts-Community/ModernFlyouts/blob/main/docs/GSMTC-Support-And-Popular-Apps.md

https://github.com/thewizrd/iTunes-SMTC
